package.json

```diff
@@ -29,6 +29,9 @@
 		"extends": "@steebchen/commitlint-config"
 	},
 	"prettier": "@steebchen/prettier-config",
+	"dependencies": {
+		"@radix-ui/react-radio-group": "1.3.7"
+	},
```

Comment:
do not add to root packaage.json, this needs to go in the apps/ui/package.json (do via pnpm)